src/Connect/ConnectButton.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { symToStr } from "tsafe/symToStr";
+import { createComponentI18nApi } from "../i18n";
+import { cx } from "../tools/cx";
+import { fr } from "../fr";
+import { getLink } from "../link";
+
+export type ConnectButtonProps = {
+    loginUrl: string;
+    serviceName: string;
```

Comment:
This should be typed
```suggestion
    serviceName: "FranceConnect" | "AgentConnect" | "MonComptePro";
```